bindata/v4.0.0/apiservice-cabundle-controller/clusterrole.yaml

```diff
@@ -29,3 +29,21 @@ rules:
   - events
   verbs:
   - create
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - secrets
```

Comment:
I do not think this controller needs read access to all secrets in all namespaces.  It might need some, but not all.

Reply:
I was trying to get it to work to see which perms were needed, but since CI wasn't working: 6c4bf1a5c1dd2c46a0121883204997d2ec633e42